### PR TITLE
Developer

### PR DESCRIPTION
**Pull request by maya88en**

### CODE REVIEW

**Review by Ispirett — CHANGES_REQUESTED** [review]
Hey @maya88en  Really great work with project :1st_place_medal:  YOu have done a fantastic job so far.

image. 

There are just some changes to be checkout below.

#### Responsiveness 
#### For most of the images, the resolution this occurs at is at the top of the image.

**This ouccurs around 497px**
change1

change 2

chnage3

change 4

chnage 5

Friendly reminder: You should use Github/Git Flow in this project. However, changing it now would take too much time.
Please make sure you use Github/Git Flow in the next project. You will find instructions in the Patwright lesson description.

File: index.html
Comment (samgaco) [inline index.html]: For the image paths, it's better to use "Img/LOGO-NEWS.jpg" directly, call the images relative to the project path.

File: index.html
Comment (samgaco) [inline index.html]: Same here, start the path from "Img/..."

File: index.html
Comment (samgaco) [inline index.html]: Do it for all of them. Right now images are not showing if you access to the [live demo.](https://rawcdn.githack.com/maya88en/NewsWeek/f9233c619fb750a37de632d24b735ea0d408b684/index.html)

**Review by samgaco — CHANGES_REQUESTED** [review]
### Hi @maya88en 

### Good work so far :muscle: 

### You seem to have worked through the previous requirements :clap: , there are some things to work on still:

* An **horizontal scrolling bar** is showing. It should not be there.
* The **navbar should disappear** for phone and tablet screen sizes. 
* The images in your **Img** folder are not showing.
* The **header in the small screen**  seems to exceed itself in height.

### Please :pray: commit these changes and submit back for another code review when you are done. Regards!! :muscle:

File: .stickler.yml
Comment (bolah2009) [inline .stickler.yml:3]: #### `AWESOME`

Nice job using a linter. :+1: 

-----

#### `HINT`

:bulb: However the `'stylelint.config.js'` file is missing. Without this file, `.stickler.yml` won't be able to check the files.

Kindly, add the `stylelint.config.js` that was referenced above, this can be found [here](https://github.com/microverseinc/linters-config/blob/master/css/stylelint.config.js)

If you are having an issue with setting up linter, read the `Troubleshooting` section of [this repo](https://github.com/microverseinc/linters-config/tree/master/css).

File: README.md
Comment (bolah2009) [inline README.md:15]: #### `OPTIONAL`

Nice job with the descriptive readme. Screenshot of the clone can also be added. :thinking: 

To make the readme look more professional, let's consider using `markdown` syntax rather than `HTML`. 

#### Find more details on using markdown [here](https://guides.github.com/features/mastering-markdown/)

File: style.css
Comment (bolah2009) [inline style.css]: #### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

File: style.css
Comment (bolah2009) [inline style.css]: Check the comment on deleting commented out code.

File: style.css
Comment (bolah2009) [inline style.css]: Check the comment on deleting commented out code.

File: index.html
Comment (bolah2009) [inline index.html]: #### `HINT`

Using bootstrap classes, let's make this a `display: flex` for medium to large screens, justify it's content to center and making all its children (or items) have equal `flex-grow`, say `1` and align text to center. This will help with making the navigation content properly centered. See the current comparison.

| Clone | Source |
| :-----: | :-----: |
| newsweek lg nav clone  | newsweek lg nav source   |

File: style.css
Comment (bolah2009) [inline style.css]: #### `HINT`

Consider making the media query screens around `990px` instead. This will prevent the large navbar from displaying inappropriately. See below:


| Clone | Source |
| :-----: | :-----: |
| newsweek lg res nav clone  | newsweek lg res nav source   |

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Awesome Job @maya88en! :muscle: :sunglasses: 

The clone looks good to me and meets *most* requirements, a few changes before :+1:, check the comments below.

- Add appropriate linter file. [HINT](https://github.com/maya88en/NewsWeek/pull/1#discussion_r315027177)
- Make sure **nothing is messed up**, let's work more on the responsiveness of the page. See comments for hints.
- [Next Project] Use GitHub Flow: For HTML & CSS projects, use only one feature branch (e.g. `embedding-images-and-video`, `html-forms`, `responsive-design`, etc) and **do not merge any feature branch to master until it's been approved**.

##### I will follow-up with a message on slack too. :+1: 

Keep up the good work and best of luck on the next submission. :+1:

Happy Coding!  :dizzy: and Have a Nice Day. :smile:

Cheers
[Bola](https://bolabuari.com/)

###### Note that comments tagged `OPTIONAL` are strongly recommended but not required to approve the project.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Hi Maya :wave: 

Nice job implementing the changes. There are still some issues with the breakpoints. Kindly fix this and submit for another code review.

You can recreate this issue by visiting [the clone](https://raw.githack.com/maya88en/NewsWeek/developer/index.html) and **inspect** with `Ctrl + shift + I` and **Toggle device toolbar** with `Ctrl + shift + M` on chrome browser.


Kindly do this and submit for another code review request.

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

File: style.css
Comment (bolah2009) [inline style.css]: #### `HINT`

:bulb: Let's make this `display: none` instead. Element with `visibility: hidden` occupied still occupy its space with the exception of not showing. Using  `visibility: hidden` will keep the logo from rendering on the far left of the page as in the source page.

```suggestion
.header-top .date { display: none; margin-top: 1px}
```

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Hi @maya88en  good job on the implementations so far,  everything looks code except for space at the right between the opinion section and the container.

Currently, stickler is not working, did you activating on this repository? :thinking: 
validate the HTML too, one error escaped your watch :smile: 
Please fix this and submit for re-review

**Review by uche-inyama — CHANGES_REQUESTED** [review]
Hi there, everything looks almost great, :clap:  But in your ``` editor's pick section ``` the tags on each image doesn't align correctly. Please fix this, and submit a code re- review. You are almost there :muscle:

**Review by uche-inyama — CHANGES_REQUESTED** [review]
Hi there everything looks almost great, :clap: But in your editor's pick section the tags on each image doesn't align correctly. Please fix this, and submit a code re- review. You are almost there :muscle:

#### Sorry about this comment, it is the same as the previous one.

**Review by bolah2009 — APPROVED** [review]
## It's a :+1: from me Maya 

Fix linter errors before merging.

**Pull request merged**
